src/components/Breadcrumb/Breadcrumb.module.css

```
@@ -0,0 +1,22 @@
+.BreadcrumbContainer {
```

⛏️ Because css modules prefix the class with the module name, we've begun to adopt simpler, context-specific class names. For instance, this one would become:
```suggestion
.container {
```